### ISSUE TITLE
logs not being generated from c7n-org 

### ISSUE THREAD

**Issue by pwatt** [capitalone/cloud-custodian#2674]
I have the following policy saved as s3.yaml:

```
policies:
  - name: cloud-custodian-s3-encryption-policy
    # ensure s3 buckets are encrypted
    resource: s3
    actions:
      - set-bucket-encryption

  - name: cloud-custodian-s3-delete-global-grants-policy
    # global grants are not allowed
    resource: s3
    filters:
      - type: global-grants
    actions:
      - delete-global-grants
```

Executing with the following command:
`c7n-org run -c accounts.yml -s s3://my-bucketname -u global/s3.yaml --region all`

The log file (custodian-run.log.gz) that are generated are empty.
[custodian-run.log.gz](https://github.com/capitalone/cloud-custodian/files/2233536/custodian-run.log.gz)

**Comment by thisisshi** [comment]
so does c7n-org itself post to stdout? can you post a sanitized version of those logs?

**Comment by pwatt** [comment]
Here's an example of what c7n-org is generating for stdout

`2018-07-26 18:30:37,352: c7n_org:INFO Ran account:my-account region:ca-central-1 policy:cloud-custodian-s3-encryption-policy matched:28 time:64.76
2018-07-26 18:30:38,145: c7n_org:INFO Ran account:my-account region:ap-northeast-2 policy:cloud-custodian-s3-encryption-policy matched:28 time:65.59
2018-07-26 18:30:38,504: c7n_org:INFO Ran account:my-account region:eu-west-1 policy:cloud-custodian-s3-encryption-policy matched:28 time:65.90
2018-07-26 18:30:40,218: c7n_org:INFO Ran account:my-account region:eu-central-1 policy:cloud-custodian-s3-encryption-policy matched:28 time:67.63
2018-07-26 18:30:40,331: c7n_org:INFO Ran account:my-account region:ap-northeast-1 policy:cloud-custodian-s3-encryption-policy matched:28 time:67.77
2018-07-26 18:30:43,101: c7n_org:INFO Ran account:my-account region:ap-south-1 policy:cloud-custodian-s3-encryption-policy matched:28 time:70.52
2018-07-26 18:30:43,173: c7n_org:INFO Ran account:my-account region:ap-southeast-1 policy:cloud-custodian-s3-encryption-policy matched:28 time:70.60
2018-07-26 18:30:45,112: c7n_org:INFO Ran account:my-account region:ap-southeast-2 policy:cloud-custodian-s3-encryption-policy matched:29 time:72.53
2018-07-26 18:32:35,273: c7n_org:INFO Ran account:my-account region:eu-west-2 policy:cloud-custodian-s3-encryption-policy matched:28 time:66.85
2018-07-26 18:32:38,797: c7n_org:INFO Ran account:my-account region:eu-west-3 policy:cloud-custodian-s3-encryption-policy matched:28 time:66.03
2018-07-26 18:32:41,973: c7n_org:INFO Ran account:my-account region:us-east-2 policy:cloud-custodian-s3-encryption-policy matched:28 time:61.39
2018-07-26 18:32:43,742: c7n_org:INFO Ran account:my-account region:sa-east-1 policy:cloud-custodian-s3-encryption-policy matched:29 time:66.76
2018-07-26 18:32:44,269: c7n_org:INFO Ran account:my-account region:us-east-1 policy:cloud-custodian-s3-encryption-policy matched:28 time:64.79
2018-07-26 18:32:47,820: c7n_org:INFO Ran account:my-account region:us-west-2 policy:cloud-custodian-s3-encryption-policy matched:28 time:61.97
2018-07-26 18:32:49,186: c7n_org:INFO Ran account:my-account region:us-west-1 policy:cloud-custodian-s3-encryption-policy matched:28 time:63.40
2018-07-26 18:33:34,613: c7n_org:INFO Policy resource counts Counter({'cloud-custodian-s3-encryption-policy': 422, 'cloud-custodian-s3-delete-global-grants-policy': 0})
`

**Comment by pwatt** [comment]
Any updates?  Is there any additional information that is required?

**Comment by thisisshi** [comment]
@pwatt thanks for the bug report, i've confirmed that this isn't working on trunk, I'll try to find some time to look into this

**Comment by jdbeitz** [comment]
Hello - looking for an update on this as well.

**Comment by kapilt** [comment]
was poking around at this, c7n-org does not support cloud watch logs for accounts executions at the moment (open question on log stream naming and also which account for the log group (target or master/origin). c7n-org also explicitly squelches the log output on custodian to warning/error to keep the cli output clean.  we should separate the log levels filtering for the log file in output from the cli output to allow for this differential handling which would address the use case in this issue.

**Comment by kapilt** [comment]
also worth noting that s3 policies are generally better off pinned to a region, as they are discoverable globally, ie you'll get additional reports for the same buckets in each region.

**Comment by kapilt** [comment]
added a fix for this to #3003
